requirements_dev.txt

```diff
@@ -1,9 +1,15 @@
 autopep8
 doc8
 flake8
+flake8-blind-except
+flake8-builtins
+flake8-docstrings
+flake8-import-order
+flake8-rst-docstrings
```

Comment:
Should we also use those in a fixed version like pylint and use the local version for the pre commit hooks?

Reply:
Currently not needed for `pre-commit`. In fact, one wonders whether `flake8` c.s. needs to be listed under `requirements_dev.txt` at all, although one would need that for e.g. IDEs.

Reply:
Wouldn't that be good to have consistent versions for pre commit and the IDE?

Reply:
Okay https://github.com/ComPWA/pycompwa/pull/77/commits/f32a2ad78c8d0f653c2d0b1c63a6fbffd1daf628

Reply:
ok